rotkehlchen/tests/exchanges/test_coinbase.py

```diff
@@ -860,6 +860,218 @@ def test_asset_conversion_choosing_fee_asset():
     assert trade == expected_trade
 
 
+def test_coinbase_query_trade_history_advanced_fill(function_scope_coinbase):
+    """Test that coinbase trade history query works fine for advanced_trade_fill"""
+    coinbase = function_scope_coinbase
+
+    mock_transactions_response = """
+{ "data": [{
+            "id": "REDACTED",
+            "type": "advanced_trade_fill",
+            "status": "completed",
+            "amount": {
+                "amount": "-192.790000",
+                "currency": "USDC"
+            },
+            "native_amount": {
+                "amount": "-176.94",
+                "currency": "EUR"
+            },
+            "description": null,
+            "created_at": "2024-03-07T08:12:51Z",
+            "updated_at": "2024-03-07T08:12:51Z",
+            "resource": "transaction",
+            "resource_path": "/v2/accounts/REDACTED/transactions/REDACTED",
+            "instant_exchange": false,
+            "advanced_trade_fill": {
+                "fill_price": "0.9174",
+                "product_id": "USDC-EUR",
+                "order_id": "REDACTED",
+                "commission": "0",
+                "order_side": "sell"
+            },
+            "details": {
+                "title": "Filled USDC on sell",
+                "subtitle": null,
+                "header": null,
+                "health": "positive"
+            },
+            "hide_native_amount": false
+        },
```

Comment:
can you add the other side of a buy/sell to check that it correctly gets skipped? Also to have it documented with an example